packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -105,6 +109,15 @@ SelfUtils.getSelves = (oldSelf, newSelf, deviceId) => {
 */
 SelfUtils.isJoined = (self) => self?.state === _JOINED_;
 
+/**
+ * Validate if the Meeting Layout Controls Layout has changed.
+ *
+ * @param {Self} previous - Previous self state
+ * @param {Self} current - Current self state [per event]
+ * @returns {boolean} - If the MEeting Layout Controls Layout has changed.
+ */
+SelfUtils.layoutChanged = (previous, current) => current?.layout && previous?.layout !== current?.layout;
```

Comment:
It will determine that the layout "has not changed", since the new self object did not include a `layout` key:value pair.

In the case that there is a `new` `layout` key:value pair [provided by `current`], it checks to see if it is a different value compared to the `previous` `layout` key:value.